Index: src/main/java/com/example/CompanyB/TrainingSimulationPrototypingModule/DTO/SimTO.java
===================================================================
diff --git a/src/main/java/com/example/CompanyB/TrainingSimulationPrototypingModule/DTO/SimTO.java b/src/main/java/com/example/CompanyB/TrainingSimulationPrototypingModule/DTO/SimTO.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/CompanyB/TrainingSimulationPrototypingModule/DTO/SimTO.java	
@@ -0,0 +1,4 @@
+package com.example.CompanyB.TrainingSimulationPrototypingModule.DTO;
+
+public class SimTO {
+}
